.github/workflows/release.yml

```
@@ -81,13 +76,45 @@ jobs:
         ./bin/lima-and-qemu.pl
         mv src/lima/lima-and-qemu.tar.gz src/lima/lima-and-qemu.macos.tar.gz
         sha512sum src/lima/lima-and-qemu.macos.tar.gz > src/lima/lima-and-qemu.macos.tar.gz.sha512sum
+
     - name: Upload MacOS build
       uses: actions/upload-artifact@v2
       with:
         name: lima-and-qemu.macos
         path: ./src/lima/lima-and-qemu.macos*
         if-no-files-found: error
 
+  macos-ventura-build:
+    runs-on: macos-13
+    timeout-minutes: 60
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.19.x
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+        submodules: recursive
+        persist-credentials: false
+
+    - name: Remove pre-installed Colima and Lima
+      run: brew uninstall colima lima
+
+    - name: Make and install Lima
+      run: make -C src/lima all install
+
+    - name: Create Ventura limactl tarball
+      run: |
+        ./bin/limactl.pl ventura
```

This feels like overkill. You shouldn't need a script to create the tarball. Untested, but something like this should work (and then you can delete the Perl script):

```suggestion
    - name: Make Lima and create Ventura tarball of limactl
      run: |
        make -C src/lima all
        tar cfz src/lima/limactl.ventura.tgz -C src/lima/_output/bin limactl
```

---

> This feels like overkill. You shouldn't need a script to create the tarball. Untested, but something like this should work (and then you can delete the Perl script):
> 
> 
> 
> ```suggestion
> 
>     - name: Make Lima and create Ventura tarball of limactl
> 
>       run: |
> 
>         make -C src/lima _output/bin/limactl
> 
>         tar cfz src/lima/limactl.ventura.tgz -C src/lima/_output/bin limactl
> 
> ```

Ack I will just do a command.

---

Not sure if this will be necessary, but seem prudent to add (between `make` and `tar`):

```
chmod u+w src/lima/_output/bin/limactl
```